0.22.5 broke typescript

## Summary

0.22.5 broke Typescript.
https://github.com/rebeccahughes/react-native-device-info/pull/478/files

## Current behavior

```
src/containers/App/index.tsx:123:5 - error TS2554: Expected 1 arguments, but got 0.

123     DeviceInfo.isPinOrFingerprintSet()((isPinOrFingerprintSet: boolean) =>
```

## Expected behavior

Not throw an error.

--- a/deviceinfo.d.ts
+++ b/deviceinfo.d.ts
@@ -27,7 +27,7 @@ declare const _default: {
   isTablet: () => boolean;
   getFontScale: () => number;
   is24Hour: () => boolean;
-  isPinOrFingerprintSet: (cb: (isPinOrFingerprintSet: boolean) => void) => void;
+  isPinOrFingerprintSet(): (cb: (isPinOrFingerprintSet: boolean) => void) => void;
   hasNotch: () => boolean;
   getFirstInstallTime: () => number;
   getLastUpdateTime: () => number;

